broker/Gemfile

```diff
@@ -11,6 +11,7 @@ gem 'systemu'
 gem 'mongoid'
 gem 'bson'
 gem 'bson_ext'
+gem 'net-ssh'
```

Comment:
Can you confirm that this rubygem is packaged as an rpm and is being installed on the broker already? Also, we'll need a corresponding change for OpenShift Online broker configuration.